src/libcmd/repl.cc

```diff
@@ -470,6 +472,10 @@ ProcessLineResult NixRepl::processLine(std::string line)
         loadFlake(arg);
     }
 
+    else if (command == ":ll" || command == ":last-flake") {
```

Comment:
ll = `last loaded`. The name `:show-scope` would be misleading I believe, since we're not actually showing the global scope, just the items that were most recently loaded. `:last-scope` or `:ls` might be a bit too close to the program `ls` and can cause confusion.
`:ll` applies to anything that loads an attr set into global scope, that is `:a`, `:l`, `:lf`, and repl cmdline args.